### ISSUE TITLE
Deno @1.5.0: TS2345 [ERROR]: Argument of type 'BufReader' is not assignable to parameter of type 'string | BufReader'

### ISSUE THREAD

**Issue by WorrenGilgamesh** [denoland/deno#8210]
I'm exploring Deno with cvs files, but i got a problem with the parse. 
I'm think that the problem come with the old dependencies;
```
 deno info --unstable --no-check .\mod.ts
``` 
The majority are from @0.75 and I'm in the @1.5. <br> 

I'm using:

    deno 1.5.0
    v8 8.7.220.3
    typescript 4.0.

I'm having having issue with this code:

```deno
import { join } from "https://deno.land/std/path/mod.ts";
import { parse } from "https://deno.land/std/encoding/csv.ts";
import { BufReader } from "https://deno.land/std/io/bufio.ts";

async function loadData() {
    const path = join(".", "data.csv");
  
    const file = await Deno.open(path);
    const bufReader = new BufReader(file);
    
    const result = await parse(bufReader, {
        skipFirstRow: true,
        comment: "#",
    });
    Deno.close(file.rid)
    console.log(result)
}

await loadData();
```
The error:

```bash
error: TS2345 [ERROR]: Argument of type 'BufReader' is not assignable to parameter of type 'string | BufReader'.
Type 'import("https://deno.land/std/io/bufio.ts").BufReader' is not assignable to 
type 'import("https://deno.land/std@0.75.0/io/bufio.ts").BufReader'.
    Types have separate declarations of a private property 'buf'.
  const result = await parse(bufReader, {
                             ~~~~~~~~~
    at...
```

In Deno 1.0.0 this similar code run well

```deno
import { join } from "https://deno.land/std/path/mod.ts";
import { parse } from "https://deno.land/std/encoding/csv.ts";
import { BufReader } from "https://deno.land/std/io/bufio.ts";

async function loadData() {
    const path = join(".", "data.csv");
  
    const file = await Deno.open(path);
    const bufReader = new BufReader(file);
    
    const result = await parse(bufReader, {
        header: true,
        comment: "#",
    });
    Deno.close(file.rid)
    console.log(result)
}

await loadData();
```

**Comment by lucacasonato** [comment]
This issue should be fixed in 1.5.1 which was just released.

**Comment by WorrenGilgamesh** [comment]
Thks!!!
